app/components/two_column_layout/two_column_layout.css

```diff
@@ -9,7 +9,7 @@
   overscroll-behavior: contain;
 
   width: 20rem;
-  height: calc(100vh - var(--nav-bar-height));
+  height: calc(100vh - 2 * var(--nav-bar-height));
```

Comment:
this fixes a bug where the footer is not displayed on the contributor's show page because it only accounted for the height of the header, not the height of the footer.